### DISCUSSION TITLE
Install this extension without using the UI?

### DISCUSSION THREAD

**Discussion by whydna** [d8ahazard/sd_dreambooth_extension#925] [Q&A] [answered]
Is it possible to install this extension to sdwebui without actually using the UI? For example, by manually building this extension and copying contents into the extensions folder for sdwebui?

**Comment by Arilziem** [comment, marked as answer]
Yes, you can download a zip here on github (or use git clone) and drop the folder into sdwebui > extensions. The webui installer doesn't do much more than that anyway afaik (at least in this case).

You can then use the webui-user-dreambooth.sh/.bat to start everything (after copying it into the main folder). That should install all necessary dependencies and then start up normally.

**Reply by whydna** [reply]
tnx!

**Comment by polym** [comment]
Can Dreambooth be run without the WebUI? If so, I would like to train models with various parameters using a bash script.

**Reply by Matthieubmt** [reply]
i have the same request

**Reply by klubkelli** [reply]
So do I. Like do you need Automatic111 or can you just run it on its own?
